modules/serverless-logic-persistence-db-schema-initialization.adoc

```diff
@@ -0,0 +1,104 @@
+// Module included in the following assemblies:
+// * serverless-logic/serverless-logic-managing-persistence
+
+
+:_mod-docs-content-type: REFERENCE
+[id="serverless-logic-persistence-db-schema-initialization_{context}"]
+= Database schema initialization
+
+When you are using `SonataFlow` with PostgreSQL persistence, you can initialize the database schema either by enabling Flyway or by manually applying database schema updates using DDL (Data Definition Language) scripts.
+
+Flyway is managed by `kie-addons-quarkus-flyway` runtime module and it is disabled by default. To enable Flyway, you must configure it using one of the following methods:
+
+[id="serverless-logic-flyway-config-workflow-configmap_{context}"]
+== Flyway configuration in the workflow ConfigMap
+
+To enable Flyway in the workflow ConfigMap, you can add the following property:
+
+.Example of enabling Flyway in the workflow ConfigMap
```

Comment:
```suggestion
.Example of enabling Flyway in the workflow `ConfigMap`
```

Later on line no 33, workflow is capitalized as "Workflow" and line no 37 mentions it as "workflow" 
- Suggestion to consider using _workflow vs Workflow_ consistently.